app/models/katello/glue/pulp/repo.rb

```diff
@@ -744,6 +744,13 @@ def ostree?
         self.content_type == Repository::OSTREE_TYPE
       end
 
+      def capsule_download_policy
+        if self.is_a?(::Katello::Repository)
+          repo = self.library_instance || self
+          repo.download_policy
+        end
+      end
```

Comment:
Moving this to the repository model will make it unavailable to content view puppet environments, which we need. I can move the function back to dynflow, but that will make it hard to test. Not sure where this should live :/